docs/data/sdks/next-gen-sdks.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Next Generation SDKs
+description: Introduction to Next Generation SDK, common methods and benefits
+---
+
+This article helps you remove confusion around various Amplitude Analytics SDKs and introduces the Next Generation SDKs.
+
+The **Next Generation (next-gen) SDKs** is a group of Amplitude Analytics SDKs that share the same architecture and interfaces. 
+
+Amplitude puts an SDK into maintenance status and under the "**Maintenance SDKs**" section in the left bar if another same-platform next-gen SDK is available, for example, Browser, Android, React Native, and Node.js. Maintenance SDKs only receives bug fixes until deprecation. It's strongly recommended to adopt next-gen SDKs to take advantage of their benefits. To migrate from a maintenance SDK, refer to the migration guide under each next-gen SDK documentation. 
+
+!!!note 
+    There are still some SDKs that aren't nex-gen while also not in maintenance, for example Flutter, Java, and iOS. 
+
+Refer to the [SDK status table](./#sdk-status-table) to check whether a SDK is next-gen or in maintenance. 
+
+## Architecture
+
+Next-gen SDKs share the same architecture and interfaces across platform. Refer to each SDK full documentation for more details. 
+
+![Next-gen Architecture](/../assets/images/data-next-gen-sdk-architecture.png)
```

Comment:
Consider using drawio to create the diagrams as a `.drawio.svg`. See the experiment docs as an example.

If you install the drawio plugin in vscode you can create and edit images in your IDE while developing locally. The special file format lets you edit the file directly, so you dont need to store a local editable version and a png for the site and manage the two separately. Even better you can save your diagram, refresh the local server and see the updated image. 

![Screen Shot 2023-03-30 at 9 23 50 AM](https://user-images.githubusercontent.com/3282199/228901781-70100219-6d60-4d17-b094-0d1e977c1b73.png)

All the images I manage for experiment docs use this format and its very nice when I need to edit them. See docs/assets/images/experiment for examples.

Reply:
@bgiori This is awesome! Also added this tool to the Wiki https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/wiki#4-optional-but-highly-encouraged-install-vscode-drawio